builds/overview-openshift-builds.adoc

```diff
@@ -0,0 +1,41 @@
+:_content-type: ASSEMBLY
+[id="overview-openshift-builds"]
+= Overview of {builds-title}
+:context: overview-openshift-builds
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+
+{builds-title} is an extensible framework that you can use to build container images on the {ocp-product-title} cluster. The framework supports tools, such as Kaniko, Cloud Native Buildpacks, Buildah, and others and is based on the following four elements:
+
+* Source code: Defines the source code that you will build
+* Output image: Defines the location to push your application images 
+* Build strategy: Defines the strategy to assemble your application 
+* Invocation: Defines the time when you want to build your application
```

Comment:
If you're interested in reducing the number of bulleted lists, a [description list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/#basic-description-list) might work here:
```suggestion
Source code:: Defines the source code that you will build
Output image:: Defines the location to push your application images 
Build strategy:: Defines the strategy to assemble your application 
Invocation:: Defines the time when you want to build your application
```